src/metpy/plots/declarative.py

```diff
@@ -647,6 +651,35 @@ class MapPanel(Panel):
     This trait sets a user-defined title that will plot at the top center of the figure.
     """
 
+    @validate('area')
+    def _valid_area(self, proposal):
+        """Check that proposed string or tuple is valid and turn string into a tuple extent."""
+        area = proposal['value']
+
+        # Parse string, check that string is valid, and determine extent based on string
+        if isinstance(area, str):
+            region, modifier = self._parse_area_string(area)
+            if region.lower() not in _areas.keys() and region != 'global':
+                raise TraitError('"' + area + '" is not a valid string area.')
```

Comment:
Would be good to add a test or two that checks that we get the expected error when we give it a bad area.